assets/src/components/progressBar/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Progress bar for displaying visual feedback about steps-completed.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+/**
+ * Progress bar.
+ */
+class ProgressBar extends Component {
+
+	/**
+	 * Get completion as a percentage.
+	 *
+	 * @return int
+	 */
+	getCompletionPercentage() {
```

Comment:
Suggestion: instead of getting these values from props rewrite the function to take two parameters (completed, total) and return a value. It would be more easily testable and allows for sanitization to occur in only one place (more on this in a separate comment).